catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -265,8 +274,12 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
                     // Prevent adding a field already on document
                     if (solrInputDocument.getFieldValue(formatIndexName) == null) {
                         solrInputDocument.addField(formatIndexName, attributeValues);
-                        if (!(formatIndexName.endsWith(SchemaFields.BINARY_SUFFIX)
-                                || formatIndexName.endsWith(SchemaFields.OBJECT_SUFFIX))) {
+                        if (AttributeFormat.GEOMETRY.equals(format)) {
+                            solrInputDocument.addField(
+                                    formatIndexName + SchemaFields.SORT_KEY_SUFFIX,
+                                    createCenterPoint(attributeValues));
```

Comment:
So if I understand this change correctly, it's an optimization to precalculate a sort key for the center point of the geometry. Is there any concern over loss of accuracy vs. the nearest neighbor sort, or are these functionally equivalent? What kind of performance impact will this have on bulk ingests?